src/_includes/layouts/webinar.njk

```diff
@@ -70,6 +70,27 @@ layout: layouts/base.njk
                         </div>
                     </div>
                     {% endfor %}
+                    {% if hubspot.downloadFormId %}
+                    <div class="mt-6 flex flex-col">
+                        <h3 class="mb-3">Download webinar slides:</h3>
```

Comment:
Can you please take off the colon here after "slides"

Reply:
Done!